gyro/models.py

```diff
@@ -79,8 +83,13 @@ def __load__(self, station = None, network_id = None, _id = None):
         if (station is not None and network_id is not None):
             self.data = {
                 '_id': station.get_hash(),
-                'latitude': station.latitude,
-                'longitude': station.longitude,
+                'location': {
+                    'type': 'Point',
+                    'coordinates': [
+                        station.latitude,
+                        station.longitude
```

Comment:
You probably want the reverse order here for a geojson point, see https://tools.ietf.org/html/rfc7946#page-8

Reply:
![](https://proxy.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia.giphy.com%2Fmedia%2FAjYsTtVxEEBPO%2F200.gif&f=1&nofb=1)

Fixed!